src/components/time-selector/TimeSelector.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react'
+import { checkboxArray } from './Checkboxes'
+import { ICheckbox, IDayCard } from './TimeSelectorInterfaces'
+import { TimeSelectorCard } from '../card/TimeSelectorCard'
+
+export const TimeSelector = () => {
+
+    const [cards, setCards] = useState<IDayCard[]>([])
+    const [checkboxes, setCheckboxes] = useState<ICheckbox[]>(checkboxArray)
+
+    const getCards = () => {
+        return ( 
+            cards?.map(card => {
+                const { name } = card
```

Comment:
We can avoid declaring variable here.
We can use spread operator while passing props to `TimeSelectorCards`.
`<TimeSelector {...card} />`